docs/source/quick_start.rst

```diff
@@ -140,4 +148,45 @@ A simple sampling-based training loop looks as follows:
 We use :class:`sar.DistNeighborSampler` to construct a distributed sampler and :func:`sar.DataLoader` to construct an iterator that retrurn standard local DGL blocks constructed from the distributed graph.  
 
 
-For complete examples, check the examples folder in the Git repository.
+Single-node training
+---------------------------------------------------------------------------
+Single-node training enables GNNs training on a very larg graphs on a single machine. This is achieved by running each worker as a separate process. At one time only one worker/process can store its data in a main memory (other processes keep their data saved on disk). SAR uses locks and barriers for synchronizing processes (only one process can aquire lock at a given time). When a process triggers a collective communication call, it sends the data to other processes using torch.distributed file based communication, saves its data to the disk and releases a lock. Then, the next process can load its part of the data from disk and continue its work.
```

Comment:
Minor typo: "Single-node training enables `training GNNs` (swap order of GNNs and training) on a (remove `a`) very larg`e` (missing `e`)

Reply:
fixed